src/modules/common/hocs/withError.js

```diff
@@ -0,0 +1,28 @@
+// @flow
```

Comment:
I think we can move the error handling code into an `error/` module if it has no dependencies to the app module.